build.sbt

```diff
@@ -43,6 +43,12 @@ lazy val sfsBackend = (project in backendRoot / "sfs")
   .dependsOn(gcsFileSystem)
   .dependsOn(backend % "test->test")
 
+lazy val ecsBackend = (project in backendRoot / "ecs")
```

Comment:
I think ecs is appropriate for now.  it's possible we might have other backends (and even this one might not be called ecs...)  But if we need to change it later, it wouldn't be the end of the world